sklearn/cross_decomposition/pls_.py

```diff
@@ -52,7 +54,7 @@ def _nipals_twoblocks_inner_loop(X, Y, mode="A", max_iter=500, tol=1e-06,
             x_weights = np.dot(X_pinv, y_score)
         else:  # mode A
             # Mode A regress each X column on y_score
-            x_weights = np.dot(X.T, y_score) / np.dot(y_score.T, y_score)
+            x_weights = np.dot(X.T, y_score)
```

Comment:
If I understand weel, this line caused y to be always scaled not matter of the parameter, causing
#6002, right ?